app/frontend/src/features/profile/edit/EditProfilePage.test.tsx

```diff
@@ -21,6 +21,15 @@ jest.mock("components/MarkdownInput");
 const getUserMock = service.user.getUser as jest.MockedFunction<
   typeof service.user.getUser
 >;
+
+const getLanguagesMock = service.resources.getLanguages as jest.MockedFunction<
```

Comment:
I didn't know about `jest.MockedFunction` type exists! Looks like jest has the same idea as me when I wrote the `MockedService` helper type (maybe as a low priority task we can gradually remove it in the future, not super important I guess)